mobileapps/tests.py

```diff
@@ -383,3 +383,134 @@ def test_mobileapps_users_remove(self):
         self.assertEqual(response.data['count'], 2)
         self.assertEqual(len(response.data['results']), 2)
         self.assertEqual(response.data['num_pages'], 1)
+
+
+@ddt.ddt
+class MobileappsOrganizationApiTests(ModuleStoreTestCase, APIClientMixin):
+    """ Test suite for Mobileapps User API views """
+
+    MODULESTORE = TEST_DATA_SPLIT_MODULESTORE
+
+    def setUp(self):
+        super(MobileappsOrganizationApiTests, self).setUp()
+
+        self.test_mobileapp_name = str(uuid.uuid4())
+        self.test_mobileapp_identifier = str(uuid.uuid4())
+        self.test_mobileapp_operating_system = 1
+        self.test_mobileapp_current_version = str(uuid.uuid4())
+        self.organizations = [
+            Organization.objects.create(name='ABC Organization'),
+            Organization.objects.create(name='XYZ Organization'),
+        ]
+
+        self.mobileapp = self._setup_test_mobileapp()
+
+        self.user = UserFactory.build(username='test', email='test@edx.org', password='test_password')
```

Comment:
this statement should be enough to create a user
```
UserFactory.create(username='test', email='test@edx.org', password='test_password')
```
we don't need to  call `save` explicitly.